src/test/java/ioananadroflorea/tests/Tema5/DemoQA.java

```diff
@@ -0,0 +1,111 @@
+    }
+
+    public static void checkBox(){
+        ChromeDriver driver = BrowserManager.createChromeDriver();
+        driver.get("https://demoqa.com/elements");
+        WebElement checkBox = driver.findElement(By.id("item-1"));
+        checkBox.click();
+        WebElement clickBoxHome = driver.findElement(By.cssSelector("span.rct-title"));
+        clickBoxHome.click();
+        WebElement textAfisat = driver.findElement(By.id("result"));
+        textAfisat.getText();
+        System.out.println(textAfisat.getText());
+        WebElement clickPlus = driver.findElement(By.cssSelector("#tree-node > div > button.rct-option.rct-option-expand-all"));
+        clickPlus.click();
+        WebElement clickMinus = driver.findElement(By.cssSelector("#tree-node > div > button.rct-option.rct-option-collapse-all"));
+        clickMinus.click();
+        driver.quit();
+    }
+    public static void radioButton(){
+        ChromeDriver driver = BrowserManager.createChromeDriver();
+        driver.get("https://demoqa.com/elements");
+        WebElement radioButton = driver.findElement(By.id("item-2"));
+        radioButton.click();
+        WebElement radioButtonAnswer = driver.findElement(By.id("impressiveRadio"));
+        radioButtonAnswer.click();
+        driver.quit();
+    }
+
+    public static void webTables(){
+        ChromeDriver driver = BrowserManager.createChromeDriver();
+        driver.get("https://demoqa.com/elements");
+        WebElement webTables = driver.findElement(By.id("item-3"));
+        webTables.click();
+        WebElement addButton = driver.findElement(By.id("addNewRecordButton"));
+        addButton.click();
+        WebElement firstName = driver.findElement(By.id("firstName"));
+        firstName.click();
+        firstName.clear();
+        firstName.sendKeys("John");
+        WebElement lastName = driver.findElement(By.id("lastName"));
+        lastName.click();
+        lastName.clear();
+        lastName.sendKeys("Doe");
+        WebElement email = driver.findElement(By.id("userEmail"));
+        email.click();
+        email.clear();
+        email.sendKeys("johndoe@yahoo.com");
+        WebElement age = driver.findElement(By.id("age"));
+        age.click();
+        age.clear();
+        age.sendKeys("22");
+        WebElement salary = driver.findElement(By.id("salary"));
+        salary.click();
+        salary.clear();
+        salary.sendKeys("2200");
+        WebElement department = driver.findElement(By.id("department"));
+        department.click();
+        department.clear();
+        department.sendKeys("Management");
+        WebElement submitButton = driver.findElement(By.id("submit"));
+        submitButton.click();
+        driver.quit();
+    }
+
+    public static void button(){
+        ChromeDriver driver = BrowserManager.createChromeDriver();
+        driver.get("https://demoqa.com/elements");
+        WebElement buttons = driver.findElement(By.xpath("//*[@id=\"item-4\"]"));
```

Comment:
Try to remove the double quotes with single quotes inside strings in order to avoid Java escape character. Applicable to all occurrences. As an example "//[@id="item-4"]" will become "//[@id='item-4']". Thanks!